packages/stark-ui/src/modules/collapsible/components/collapsible.component.ts

```diff
@@ -0,0 +1,98 @@
+import { Component, EventEmitter, HostBinding, Inject, Input, OnInit, Output } from "@angular/core";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+import { StarkCollapsibleOptions } from "@nationalbankbelgium/stark-ui";
+
+/**
+ * Name of the component
+ */
+const componentName: string = "stark-collapsible";
+
+/**
+ * Component to display an accordion around included html using angular material's expansion panel.
+ */
+@Component({
+	selector: "stark-collapsible",
+	templateUrl: "./collapsible.component.html"
+})
+export class StarkCollapsibleComponent implements OnInit {
+	/**
+	 * Adds class="stark-app-logo" attribute on the host component
+	 */
+	@HostBinding("class")
+	public class: string = componentName;
+
+	/**
+	 * ID to set to the collapsible
+	 */
+	@Input()
+	public htmlId: string;
+
+	/**
+	 * Icon to use in the header (based on Material Design Icons)
+	 */
+	@Input()
+	public icon: "chevron-right" | string = "chevron-right";
+
+	/**
+	 * Boolean value triggering the collapsing of the collapsible
+	 */
+	@Input()
+	public isExpanded: boolean = false;
+
+	/**
+	 * Output value giving the collapse state of the collapsible
+	 */
+	@Output()
+	public isExpandedChange: EventEmitter<boolean> = new EventEmitter();
+
+	/**
+	 * String or translation key that will be displayed in the title part of the header.
+	 */
+	@Input()
+	public titleLabel: string;
+
+	/**
+	 * String or translation key that will be displayed in the sub-title part of the header.
+	 */
+	@Input()
+	public subTitleLabel?: string;
```

Comment:
I don't think we should add a subtitle to the collapsible panel, I think the collapsible header could become a bit messy